StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/NoXmlFileHeaderUnitTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.DocumentationRules;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// Unit tests for file header that do not follow the XML syntax.
+    /// </summary>
+    public class NoXmlFileHeaderUnitTests : DiagnosticVerifier
+    {
+        private const string SettingsFileName = "stylecop.json";
+        private const string TestSettings = @"
+{
+    ""settings"": {
+    ""documentationRules"": {
+      ""copyrightText"": ""Copyright (c) FooCorp. All rights reserved.\nLicensed under the ??? license. See LICENSE file in the project root for full license information."",
+      ""xmlHeader"": false
```

Comment:
I added this in commit a4427f016659dcff4ef6047586753a3dd7b62ca7 and then merged the result. :+1: